### DISCUSSION TITLE
Welcome to Metals discussions

### DISCUSSION THREAD

**Discussion by gabro** [scalameta/metals#2300] [General]
Hello! 👋

We're testing out the newly available GitHub Discussions for Q&A issues. Previously we had a "question" label for issues (and an issue template for it) whereas now those questions should be asked a new GitHub discussions (and the new issue template reflects that).

GitHub Discussions are particularly appealing for Q&A and support requests in general because:
- they can be more focused than a chat, especially in case of multiple threads of discussion happening at the same time
- it's easier to point to previous discussions
- discussions are indexed and available in google searches (much like issues and Gitter chats)
- discussions can have answers marked as accepted (similar to StackOverflow)

For the moment, GitHub Discussion will remain a parallel way to ask questions about Metals together with Gitter and Discord.
We'll decide whether to encourage all questions to be posted there once we gather more experience with this new GitHub feature (it will reasonably take a while).
Feel free to ask questions wherever it suits you, but asking them here on GitHub will give us more material to evaluate it :-)

Thanks!

**Comment by stefanobaghino** [comment]
Nice, now I almost feel bad for pushing to have a Discourse forum for DAML. 😂
